web/api/web_api_v1.c

```diff
@@ -416,6 +417,8 @@ inline int web_client_api_request_v1_data(RRDHOST *host, struct web_client *w, c
         // name and value are now the parameters
         // they are not null and not empty
 
+        if(!strcmp(name, "context")) context = value;
```

Comment:
Is there a special motive for this if to be independent of the next `if.. else if` group?